### PR TITLE
Feedback upgrade

### PR DESCRIPTION
**Pull request by StephBrouil**
- Converted from Vue CLI to Vite which is recommended for Vue 3
- Upgraded to Vue 3
- Upgraded to Vee-Validate v4
- Switched to global localization
- Use of I18n-Host custom component as a wrapper to inject localization in Web Components
- Kept feedbackConfirmation and SubmissionInformation even though they aren't being used due to a vue-i18n limitation in web components, to switch back if the issue is addressed.
- Added textareaInput shared component

### CODE REVIEW

**Review by joshcampbell191 — CHANGES_REQUESTED** [review]
TODO: Fix issue when toggling within helpful form

**Review by joshcampbell191 — CHANGES_REQUESTED** [review]
TODO: Fix issue when toggling within helpful form

File: src/FeedbackHelpful.ce.vue [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+            <h2 class="h5">{{ t('submissionInformation.note') }}</h2>
+            <ul>
+              <li><small><strong>{{ t('contact.noReply') }}.</strong> {{ t('contact.forEnquiries') }} <a
+                    :href="`${websiteBasePath}${t('contact.opcContactLink')}`">{{ t('contact.contactOpc')
+}}</a>.</small>
+              </li>
+              <li><small><strong>{{ t('submissionInformation.noPersonalInfo') }},</strong> {{
+    t('submissionInformation.personalInfoItems')
+}}.</small></li>
+              <li><small><strong>{{ t('submissionInformation.forMoreInfo') }}</strong>, {{
```
Comment (joshcampbell191) [inline src/FeedbackHelpful.ce.vue:75]: I wonder if https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage would be cleaner to generate the `li`.

File: src/FeedbackHelpful.ce.vue [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+          </div>
+          <div>
+            <button class="btn btn-primary">{{ t('form.submit') }}</button>
+          </div>
+        </div>
+      </form>
+      <div v-if="submitSuccessful" id="feedbackSuccess">
+        <div>
+          <h2 class="h5">{{ t('confirmation.thankYouForFeedback') }}!</h2>
+          <p>{{ t('contact.noReply') }}. {{ t('contact.forEnquiries') }} <a
```
Comment (joshcampbell191) [inline src/FeedbackHelpful.ce.vue:90]: I wonder if https://kazupon.github.io/vue-i18n/guide/interpolation.html#slots-syntax-usage would be cleaner to generate the `p`.

File: src/FeedbackHelpful.ce.vue (hunk 1 above)
Comment (joshcampbell191) [reply src/FeedbackHelpful.ce.vue:75]: If you decide to move forward with this, there's a few other places that can use the same fix.

**Review by joshcampbell191 — APPROVED** [review]

File: src/FeedbackHelpful.ce.vue (hunk 1 above)
Comment (StephBrouil) [reply src/FeedbackHelpful.ce.vue:75]: It doesn't seem to be compatible with web components at the moment.

**Pull request merged**